tests/tests_suite_test.go

```diff
@@ -45,11 +46,20 @@ var _ = BeforeSuite(func() {
 
 	var err error
 	cluster.RestConfig, err = clientcmd.BuildConfigFromFlags("", *kubeconfig)
```

Comment:
Define these as globals of this suite, so we are not changing our dependencies.

Reply:
I actually need `RestConfig` in `node.execOnPod()`, that's why I wanted it external in the first place..
but maybe I can't just create another there. will try

Reply:
You can spawn a new client in each helper function. We commonly do it in KubeVirt e2e tests. It is some overhead, but it keeps the code cleaner to read, which should be the priority on tests.

Reply:
I preferred to pass the client.. let me know what you think